packages/mod.go

```diff
@@ -47,35 +44,14 @@ func (m Mod) ExtractPurlsFromManifest() (purls []string) {
 	return
 }
 
-func (m Mod) ExtractPurlsFromManifestForIQ() (purls []string) {
-	for _, s := range m.ProjectList.Projects {
-		if len(s.Version) > 0 { // There must be a version we can use
-			var version = s.Version
-			version = strings.Replace(version, "+incompatible", "", -1)
-			var purl = "pkg:" + convertGopkgNameToPurl(s.Name) + "@" + version
-			purls = append(purls, purl)
-		}
-	}
-	purls = removeDuplicates(purls)
-
-	return
-}
-
-func (m Mod) CheckExistenceOfManifest() (bool, error) {
-	if _, err := os.Stat(m.GoSumPath); os.IsNotExist(err) {
-		return false, customerrors.NewErrorExitPrintHelp(err, fmt.Sprint("No go.sum found at path: "+m.GoSumPath))
-	}
-	return true, nil
-}
-
 func removeDuplicates(purls []string) (dedupedPurls []string) {
 	encountered := map[string]bool{}
 
 	for _, v := range purls {
 		if encountered[v] {
-			LogLady.WithField("dep", v).Debug("Found duplicate dependency, eliminating it")
+			// LogLady.WithField("dep", v).Debug("Found duplicate dependency, eliminating it")
```

Comment:
Do we want to put these back??

Reply:
> Do we want to put these back??

Probably do, but now you have me thinking about the deprecated logrus issue. Not sure I should spread logrus interfaces into more method signatures. Cogitating. Worst case, might slap a @todo in these for when we deal with that.